sandbox/tests/collision/touching.ts

```diff
@@ -23,7 +23,7 @@ ball.on('update', function () {
     if (this.pos.x < (this.getWidth() / 2)) {
         this.vel.x *= -1;
     }
-    if (this.pos.x + (this.getWidth() / 2) > game.getDrawWidth()) {
```

Comment:
Could you leave these in with `@deprecated` and then move the changelog entry to 'Deprecated'?